css/card.css

```diff
@@ -101,3 +394,345 @@
 .dialog-overlay {
   background-color: rgba(43, 46, 56, 0.7);
 }
+
+/* card animation code */
+
+:root {
+  /* BASE */
+  --color1: #b8b8b8;
+  --color2: #6b6b6b;
+  /* --color1: #58abda;
+  --color2: #495173; */
+  --posx: 50%;
+  --posy: 50%;
+}
+
+.details-dialog-card, #gacha-controls .card-image {
+  width:var(--card-width); height:var(--card-height);
+  position: relative;
+  margin: 20px;
+  overflow: hidden;
+  z-index: 10;
+  touch-action: none;
+  border-radius: 5%/3.5%;
+  box-shadow: -7px -7px 10px -5px transparent, 7px 7px 10px -5px transparent, 0 0 5px 0px rgba(255, 255, 255, 0), 0 55px 35px -20px rgba(0, 0, 0, 0.5);
+  /* box-shadow: -5px -5px 5px -5px var(--color1), 5px 5px 5px -5px var(--color2), -7px -7px 10px -5px transparent, 7px 7px 10px -5px transparent, 0 0 5px 0px rgba(255, 255, 255, 0), 0 55px 35px -20px rgba(0, 0, 0, 0.5); */
+  /* box-shadow: 0 10px 20px -5px #000; */
+  transition: transform 0.5s ease, box-shadow 0.2s ease;
+  will-change: transform, filter;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+  transform-origin: center;
+  display: inline-block;
+  transform: translate3d(0.1px, 0.1px, 0.1px);
+}
+#gacha-controls .card-image {
+  box-shadow: 0 10px 20px -5px rgba(0, 0, 0, 0.5);
+}
+#gacha-controls .card-image, #gacha-controls .card-image:before, #gacha-controls .card-image:after {
+  transition: all 0.5s ease;
+}
+
+.details-dialog-card {
+  width:calc(var(--card-width)*2);
+  height:calc(var(--card-height)*2);
+}
+
+#gacha-controls .card-image {
+  margin:auto;
+}
+
+@media screen and (max-width: 600px) {
+  .details-dialog-card {
+    width:calc(var(--card-width)*1.5); height:calc(var(--card-height)*1.5);
+  }
+}
+
+.details-dialog-card.holo, #gacha-controls .card-image.holo {
+  --color1: #ec9bb6;
+  --color2: #ccac6f;
+  --color3: #69e4a5;
+  --color4: #8ec5d6;
+  --color5: #b98cce;
+}
+.details-dialog-card.ultra, #gacha-controls .card-image.ultra {
+  --color1: #ff7873c0;
+  --color2: #ffed5fc0;
+  --color3: #a8ff5fc0;
+  --color4: #83fff7c0;
+  --color5: #7894ffc0;
+}
+.details-dialog-card.secret, #gacha-controls .card-image.secret {
+  --color5: #ff787390;
+  --color4: #ffed5f90;
+  --color3: #a8ff5f90;
+  --color2: #83fff790;
+  --color1: #7894ff90;
+}
+
+.details-dialog-card:before,
+.details-dialog-card:after, 
+#gacha-controls .card-image:before,
+#gacha-controls .card-image:after {
+  content: "";
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  background-repeat: no-repeat;
+  opacity: 0;
+  mix-blend-mode: color-dodge;
+  transition: all 0.33s ease;
+  border-radius: 5%/3.5%;
+}
+
+.details-dialog-card:before, #gacha-controls .card-image:before {
```

Comment:
Add comments throughout below, particularly calling out important lines for common things to update, like higher brightness/ contrast, different texture?

Reply:
I added some comments throughout the CSS here, but I'm not sure they're as detailed as you would like.

Reply:
they're good, thank you!